services/blockchain-indexer/shared/dataService/business/blocks.js

```diff
@@ -195,7 +195,11 @@ const getBlocks = async params => {
 	const blocksTable = await getBlocksTable();
 	const blocks = {
 		data: [],
-		meta: {},
+		meta: {
+			count: 0,
+			offset: params.offset,
+			total: 0,
+		},
```

Comment:
Is this necessary?

Reply:
yes because we return from [here](https://github.com/LiskHQ/lisk-service/blob/3dc738886ae9225f760e16cd6e61f4e74034c137/services/blockchain-indexer/shared/dataService/business/blocks.js#L227)